### ISSUE TITLE
Address miscellaneous TODOs

### ISSUE THREAD

**Issue by chrisklus** [phetsims/counting-common#12]
Following https://github.com/phetsims/number-suite-common/issues/29:

This issue is a catch-all for miscellaneous TODOs that did not have an associated GitHub issue.

These TODOs should be triaged for Number Play 1.1 and Number Compare 1.0, identifying which ones need to be addressed. If any of them require significant work, I'll split out into a new issue.

**Comment by chrisklus** [comment]
Adding blocks-sim-publication label to indicate this should be completed before starting Number Compare and Number Play 1.0/1 dev testing.

**Comment by pixelzoom** [comment]
I reviewed the 12 TODOs in counting-common.  I don't see any that are essential to address before publishing Number Play and Number Compare (https://github.com/phetsims/number-compare/issues/9) -- i.e. nothing that impacts UX or crashes the sim. @chrisklus you should definitely take a look too. But I'd be OK with changing this issue from "blocks-sim-publication" to "deferred".

**Comment by zepumph** [comment]
If we were going to pull out the "ones" case in BaseNumberNode, then perhaps this patch would be helpful, but we won't, so it is all good!

<details>

```diff
Subject: [PATCH] factor out usages of "20" to a single constant, https://github.com/phetsims/counting-common/issues/12
---
Index: js/common/view/BaseNumberNode.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/common/view/BaseNumberNode.ts b/js/common/view/BaseNumberNode.ts
--- a/js/common/view/BaseNumberNode.ts	(revision c02505a2a59e03fb54b4f32587f33288ddeda1b9)
+++ b/js/common/view/BaseNumberNode.ts	(date 1680726276734)
@@ -305,34 +305,59 @@
       const width = singleCardBounds.width;
       const height = singleCardBounds.height;
 
-      const xMargin = ( width - fullObjectWidth ) * 0.5;
+      // TODO: Does number of columns matter here?
+      const xMargin = ( width - fullObjectWidth ) / 2;
       const yMargin = ( height - numberOfRows * renderedObjectHeight ) / ( numberOfRows + 1 );
+
       const yExtraMarginTop = backgroundNode.height - yMargin * ( numberOfRows + 1 ) - renderedObjectHeight * numberOfRows;
+      console.log( yExtraMarginTop );
+
+      const image = CountingCommonConstants.COUNTING_OBJECT_TYPE_TO_IMAGE.get( options.countingObjectType );
+
+      if ( value === CountingCommonConstants.ONE ) {
+        const columnWidth = width - ( 2 * xMargin );
+        const rowHeight = ( height - ( 2 * yMargin ) );
+
+
+        debugger;
+        this.addChild( new Image( image, {
+          maxWidth: renderedObjectWidth,
+          maxHeight: renderedObjectHeight,
+          // centerX: ( columnWidth / 2 ) + xMargin,
+          // centerY: ( rowHeight / 2 ) + yMargin + yExtraMarginTop,
+          // left: 0,
+          // top: 0
+          left: -singleCardBounds.minX + xMargin,
+          top: singleCardBounds.minY + yMargin + yExtraMarginTop
+        } ) );
+      }
+      else {
 
-      // add and position the object images
-      const objectImages: Image[] = [];
-      for ( let z = 0; z < numberOfSets; z++ ) {
-        for ( let i = 0; i < numberOfRows; i++ ) {
-          for ( let j = 0; j < numberOfColumns; j++ ) {
+        // add and position the object images
+        const objectImages: Image[] = [];
+        for ( let z = 0; z < numberOfSets; z++ ) {
+          for ( let i = 0; i < numberOfRows; i++ ) {
+            for ( let j = 0; j < numberOfColumns; j++ ) {
 
-            const columnWidth = ( width - ( ( numberOfColumns + 1 ) * xMargin ) ) / numberOfColumns;
-            const centerX = ( ( j + 1 ) * xMargin ) + ( j * columnWidth ) + ( columnWidth / 2 ) +
-                            // This is used to draw a second set for a double card exactly where the objects are when
-                            // stacked from a single card on a double
-                            z * ( backgroundNode.width - width + 0.1 );
+              const columnWidth = ( width - ( ( numberOfColumns + 1 ) * xMargin ) ) / numberOfColumns;
+              const centerX = ( ( j + 1 ) * xMargin ) + ( j * columnWidth ) + ( columnWidth / 2 ) +
+                              // This is used to draw a second set for a double card exactly where the objects are when
+                              // stacked from a single card on a double
+                              z * ( backgroundNode.width - width + 0.1 );
 
-            const rowHeight = ( height - ( ( numberOfRows + 1 ) * yMargin ) ) / numberOfRows;
-            const centerY = ( ( i + 1 ) * yMargin ) + ( i * rowHeight ) + ( rowHeight / 2 ) + yExtraMarginTop;
+              const rowHeight = ( height - ( ( numberOfRows + 1 ) * yMargin ) ) / numberOfRows;
+              const centerY = ( ( i + 1 ) * yMargin ) + ( i * rowHeight ) + ( rowHeight / 2 ) + yExtraMarginTop;
 
-            if ( objectImages.length < value ) {
-              const objectImage = new Image( CountingCommonConstants.COUNTING_OBJECT_TYPE_TO_IMAGE.get( options.countingObjectType ), {
-                maxWidth: renderedObjectWidth,
-                maxHeight: renderedObjectHeight,
-                centerX: centerX,
-                centerY: centerY
-              } );
-              this.addChild( objectImage );
-              objectImages.push( objectImage );
+              if ( objectImages.length < value ) {
+                const objectImage = new Image( image, {
+                  maxWidth: renderedObjectWidth,
+                  maxHeight: renderedObjectHeight,
+                  centerX: centerX,
+                  centerY: centerY
+                } );
+                this.addChild( objectImage );
+                objectImages.push( objectImage );
+              }
             }
           }
         }
```

**Comment by zepumph** [comment]
We have handled all issues linked here. Closing!
